project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject spootnik/reporter "0.1.23"
+(defproject exoscale/reporter "0.1.24"
```

Comment:
Note to reviewers: not strictly needed - however I could not release to the spootnik namespace.

Happy to change to whatever is most relevant.